packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTraffic.java

```diff
@@ -270,7 +253,7 @@ private CharSequence formatOutput(long speed) {
                 decimalFormat = new DecimalFormat("0.00");
                 formatSpeed =  decimalFormat.format(speed / (float)Giga);
             } else if (speed >= 100 * Mega) {
-                decimalFormat = new DecimalFormat("###0");
+                decimalFormat = new DecimalFormat("##0");
```

Comment:
why is # reduced here?

Reply:
# is not needed here as 1000 Mega/1 Giga will match the if (Giga).